apps/web/src/services/asset-inventory/components/MetricExplorerDataTable.vue

```diff
@@ -98,12 +103,16 @@ watch(
     async ([metricId]) => {
         if (!metricId) return;
         state.thisPage = 1;
-        const { results, more } = await analyzeMetricData();
-        state.items = getRefinedMetricExplorerTableData(results, metricExplorerPageState.granularity, metricExplorerPageState.period);
-        state.more = more ?? false;
+        await setDataTableData();
     },
     { immediate: true, deep: true },
 );
+watch(() => metricExplorerPageState.refreshMetricData, async (refresh) => {
+    if (refresh) {
+        await setDataTableData();
+        metricExplorerPageStore.setRefreshMetricData(false);
+    }
+}, { immediate: false });
```

Comment:
Question: I know default value is `false`, right..?

Reply:
Yeap...I just use it to make it clear😅